### PR TITLE
allow 10-second leeway when decoding tokens

### PR DESCRIPTION
**Pull request by MothOnMars**
Resolves https://github.com/18F/omniauth_login_dot_gov/issues/26

Note: The `leeway` option sets a global leeway, which is used for both the [not before](https://github.com/jwt/ruby-jwt#not-before-time-claim) and [expiration](https://github.com/jwt/ruby-jwt#expiration-time-claim) values:
https://github.com/jwt/ruby-jwt/blob/5818f9868ae201f123b2522679bf8af139bbba13/lib/jwt/verify.rb#L101

Those options can be passed individually, but the global leeway seemed like the simplest and most flexible solution.

### CODE REVIEW

**Comment by julialeague** [comment]
Thank you, @MothOnMars ! We'll take a look and figure out if anything needs to be added/changed and help to make those changes before merging.

I've logged https://cm-jira.usa.gov/browse/LG-8295 to track this update internally.

**Comment by MothOnMars** [comment]
@julialeague , "Allow edits from maintainers" is already enabled. Let me know if you need anything else.

**Comment by julialeague** [comment]
After merging, I'll open a follow-up PR to bump the gem version.

**Review by Jeremy1026 — COMMENTED** [review]
Looks good, just needs a bump on the gem version in `lib/omniauth/login_dot_gov/version.rb`

**Review by Jeremy1026 — APPROVED** [review]

**Pull request merged**
